### PR TITLE
[console] validate role shortname

### PR DESCRIPTION
**Pull request by tonio**
see #2781

### CODE REVIEW

**Comment by fvanderbiest** [comment]
Did I mention the org ? It was about the role IIRC.

**Comment by fvanderbiest** [comment]
Also, the regexp should only allow uppercased chars (and underscores !) for the role name.

File: console/src/main/webapp/manager/app/app.es6 [hunk 1]
```diff
@@ -145,6 +145,18 @@ angular.module('manager', [
     moment.locale($translate.use())
     return date => moment(date).format('LL')
   }])
+  .directive('shortname', () => ({
+    require: 'ngModel',
+    restrict: 'A',
+    link: (scope, elm, attrs, ctrl) => {
+      const alphanum = v => v && v.match(/^\w*$/)
```
Comment (fvanderbiest) [inline console/src/main/webapp/manager/app/app.es6]: also need to allow the underscore char (ex: role `EL_TRUITE`)

Comment (tonio) [reply console/src/main/webapp/manager/app/app.es6]: https://developer.mozilla.org/fr/docs/Web/JavaScript/Guide/Expressions_r%C3%A9guli%C3%A8res#special-word :sunglasses:

**Pull request merged**